Detectors/GlobalTrackingWorkflow/src/TOFMatcherSpec.cxx

```diff
@@ -102,6 +102,8 @@ void TOFMatcherSpec::run(ProcessingContext& pc)
   RecoContainer recoData;
   recoData.collectData(pc, *mDataRequest.get());
 
+  auto creationTime = DataRefUtils::getHeader<DataProcessingHeader*>(pc.inputs().getFirstValid(true))->creation;
```

Comment:
This I know (we are not so sensitive in our calibration).
However, this will be replaced... waiting for a standard way working both in sync and async phase.